code/datums/components/storage/concrete/bag_of_holding.dm

```diff
@@ -18,8 +18,10 @@
 			if(istype(T, /turf/open/space/transit))
 				continue
 			T.TerraformTurf(/turf/open/chasm/magic, /turf/open/chasm/magic)
-		message_admins("[key_name_admin(user)] detonated a bag of holding")
-		log_game("[key_name(user)] detonated a bag of holding")
+		var/turf/Turf = get_turf(src)
+		var/area/Area = get_area(Turf)
+		message_admins("[key_name_admin(user)] detonated a bag of holding at [Area] [ADMIN_COORDJMP(Turf)].")
+		log_game("[key_name(user)] detonated a bag of holding at [Area] [COORD(Turf)].")

```

Comment:
same

Reply:
Figures the only time I don't test something because it's simple enough is the time I fuck it up.
I did use a slightly different formatting that's more consistent with all the warnings I've done so far.